### DISCUSSION TITLE
model in with data without label

### DISCUSSION THREAD

**Discussion by oscarramirezs** [Project-MONAI/MONAI#2268] [Q&A]
Hello guys, thanks for this job ,,

my question is:
I trained a segmentation model, but my proble is that I want to use it in new data with  not  label, which is my objective of training a model (simulating the production). 
but it seems that Monai always needs a label to do the interference? even when it is in production?

Thanks you guys 

Oscar

**Comment by Nic-Ma** [comment]
Hi @oscarramirezs ,

Thanks for your interest here.
MONAI can support inference, you can refer to this tutorial example:
https://github.com/Project-MONAI/tutorials/blob/master/3d_segmentation/torch/unet_inference_dict.py

Thanks.
